### ISSUE TITLE
Several issues to be resolved in the near future

### ISSUE THREAD

**Issue by bluegill** [bluegill/katana#8]
Reminder for me to fix the many bugs and issues currently present in Katana.

* ~Unable to drag & drop images to upload; results in crash.~
* ~Upload history not showing latest results; should be an easy fix.~
* Updater module not working, resulting in a broken application for individuals not using the latest version.
* Windows support development has stalled, will continue to work on it in the near future.
* Make use of promises and asynchronous methods more.
* Several small bugs and issues that need to be addressed.

**tldr** - plan to fix current issues w/ katana ***asap*** (im a busy boy), windows support soon&trade;

**Comment by bluegill** [comment]
Drag & drop crash issue has been fixed
Upload history now shows in correct order
Windows builds now working but needs **a lot** of work before the app is fully functional.
